x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestsDataLoader.java

```diff
@@ -371,6 +371,21 @@ public static void createInferenceEndpoint(RestClient client) throws IOException
                  }
             """);
         client.performRequest(request);
+
+        request = new Request("PUT", "_inference/rerank/test_reranker");
+        request.setJsonEntity("""
+            {
+                "service": "test_reranking_service",
+                "service_settings": {
+                    "model_id": "my_model",
+                    "api_key": "abc64"
+                },
+                "task_settings": {
+                    "use_text_length": true
```

Comment:
ℹ️ Using the settings `use_text_length: true`, so the reranker score is predictable  in test instances (equal to the inverse of the length of the text sent).